akka-http/src/main/scala/akka/http/scaladsl/server/directives/FileUploadDirectives.scala

```diff
@@ -60,7 +60,10 @@ trait FileUploadDirectives {
           val uploadedF: Future[(FileInfo, File)] =
             bytes
               .runWith(FileIO.toPath(dest.toPath))
-              .map(_ ⇒ (fileInfo, dest))
+              .map { ioResult ⇒
+                val _ = ioResult.status.get
+                (fileInfo, dest)
+              }
```

Comment:
`.transform(_.status.map(_ => (fileInfo, dest)))`

Reply:
@viktorklang 
I tried it.
But that failed with ci.

Reply:
Hmm, 2.11 cross-compilation?

Reply:
I guess so.